msg/vehicle_angular_velocity.msg

```diff
@@ -1,6 +1,6 @@
 
-uint64 timestamp		# time since system start (microseconds)
+uint64 timestamp	# time since system start (microseconds)
 
-uint64 timestamp_sample		# the timestamp of the raw data (microseconds)
+uint64 timestamp_sample	# the timestamp of the raw data (microseconds)
```

Comment:
@jlecoeur looks like this isn't going to align on github without changing the tab size.

Reply:
Yes, or going for space indents